src/Main/ProjectFile/ProjectManagerComponent.cpp

```diff
@@ -266,10 +266,13 @@ void ProjectManagerComponent::Load(QXmlStreamReader* r) {
                   strFile);
             }
             int index{0};
+            auto projectPath = Project::Instance()->projectPath();
             for (const auto& i : langList) {
+              auto designFiles = i.second;
+              designFiles.replace(PROJECT_OSRCDIR, projectPath);
               projectFileset.addFiles(
                   libs.at(index).first, libs.at(index).second,
-                  ProjectManager::StringSplit(i.second, " "), i.first);
+                  ProjectManager::StringSplit(designFiles, " "), i.first);
```

Comment:
I don't think this solve the problem

Reply:
It's extending to the grouping of design files, what are we missing ?

Reply:
It doesn't solve the problem completely, I agree. It will be tackled within extended file paths exercise.
But at this point it will allow us to load *.ospr file, having relative path stored inside.